docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+env_file:
+  - path: ./kafka-env.env
+    required: true # default
+services:
+  kafka:
+    image: ghcr.io/theg-r-itters/gitassistant:kafka
+    environment:
```

Comment:
You added the values in a file and use them in the docker compose env var. It's correct but you can add the env file directly with env_file tag in docker compose file, in this way it will be cleaner

Reply:
Actually the environment file is explicitly searched and  included in the docker-compose file